### PR TITLE
Revised template files for etl

### PR DESCRIPTION
**Pull request by EricTurner3**
- added placeholder etl.py
- revised scraper to call etl.py
- revised scraper on how to call from our /common modules
- revised readme.md with note on scrapers importing etl.py

### CODE REVIEW

File: template/jurisdiction_scraper.py [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+from bs_scrapers.get_files import get_files # example of importing one of those files
+
+
+# do your code specific for the jurisdiction here, be sure to check out some other scrapers
+# you can call some of the modules in /common where some of frequently used methods are stored
+
+
+
+# make sure the end of your file calls the etl.py file!
+import etl
```
Comment (CaptainStabs) [inline template/jurisdiction_scraper.py:27]: Add newline at end of file

File: template/etl.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+'''
+ETL File
+This file will take the data dropped from the scraper: transform it and load it into dolt
+
+This is still a WIP
+'''
+  
+print("ETL File Loaded")
```
Comment (CaptainStabs) [inline template/etl.py:8]: Add newline at end of file

**Review by CaptainStabs — CHANGES_REQUESTED** [review]
Add newline at end of scripts

File: template/jurisdiction_scraper.py (hunk 1 above)
Comment (EricTurner3) [reply template/jurisdiction_scraper.py]: added in latest commit

File: template/etl.py (hunk 2 above)
Comment (EricTurner3) [reply template/etl.py]: added in latest commit

File: template/jurisdiction_scraper.py (hunk 1 above)
Comment (CaptainStabs) [inline template/jurisdiction_scraper.py:27]: Maybe I just don't see it?

**Review by CaptainStabs — APPROVED** [review]
👍 Thanks!

**Pull request merged**